README.md

```diff
@@ -1,11 +1,106 @@
-# Go Implementations of the Common Expression Language
+# Common Expression Language - Go Toolchain
 
-For background on the Common Expression Language see the [cel-spec][1] repo.
+[![Build Status](https://travis-ci.org/google/cel-go.svg?branch=master)](https://travis-ci.org/google/cel-go) [![Go Report Card](https://goreportcard.com/badge/github.com/google/cel-go)](https://goreportcard.com/report/github.com/google/cel-go)
 
+This repo contains the Go toolchain for the Common Expression Language (CEL).
+CEL is a non-Turing complete language designed to be portable and fast. It is
+best suited to  applications where sandboxing a full-fledged language like
+JavaScript or Lua would be too resource intensive, but side-effect free dynamic
+computations are strongly desired. 
+
+## Want to learn more?
+
+* See the [CEL Spec][1] for more details about the language.
+* Ask for support on the [CEL Go Discuss][2] Google group.
+
+## Want to integrate CEL?
+
+* See [![GoDoc](https://godoc.org/github.com/google/cel-go?status.svg)](https://godoc.org/github.com/google/cel-go)
```

Comment:
It looks a little counter intuitive to not have a link without an image. What would you think of:
```See [Go doc](https://godoc.org/github.com/google/cel-go) to learn how to integrate CEL into services written in Go. [![GoDoc](https://godoc.org/github.com/google/cel-go?status.svg)](https://godoc.org/github.com/google/cel-go)```

Reply:
Good suggestion.